Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_30.md

```diff
@@ -0,0 +1 @@
+## Documentation and metadata improvements.
```

Comment:
It seems you added new playbook and trigger. Those should be documented in the RN